src/renderer/pages/Wallet/WalletImport.vue

```diff
@@ -217,7 +227,7 @@ export default {
      */
     step () {
       if (this.step === 2 && !this.useOnlyAddress) {
-        this.schema.address = WalletService.getAddress(this.schema.passphrase.normalize('NFD'), this.session_network.version)
+        this.schema.address = WalletService.getAddress(this.schema.passphrase, this.session_network.version)
```

Comment:
I think this `normalize` is needed, and was added by @ItsANameToo at #779

Reply:
Yes I was pushed edits to older code. So pushed another commit 'rebase develop' that adds that thing again